guides/browser-support.md

```diff
@@ -0,0 +1,171 @@
+# Browser support
+
+> A friendly guide for building cross-browser web components
+
+## Why support?
+
+The promise of the web is to allow anyone to [access content using any browser
+on any device](https://hacks.mozilla.org/2016/07/make-the-web-work-for-everyone/).
+By building cross-browser compatible web components we ensure that our content,
+tools, and resources remain available for all that seek them, whether they be
+the American public or Bureau staff. We seek to esnure this open accessibility
+by designing and coding for unknown client environments, and continually
+testing our products in as many environments as is practical.
+
+## Tips for successfully building cross-browser components
+
+Cross-browser support doesn't come from a single place, it takes a combination
+of tools and best practices. We've outlined a few high-level topics here, but
+don't let that stop you from reaching further into your toolset. There's really
+no such thing as too much support.
+
+### User centered design
+
+We build features for our users, not to stoke our own egos. When looking at a
+new feature we first start by asking ourselves how will it benefit the user
+compared to our current offerings and what negative implications will we need
+to work around. Afterall, what good is a feature if our users are unable to
+access it. By keeping our users in the forefront of our design and development
+decisions, we can assure ourselves our work is in their best interest and ensure
```

Comment:
Something reads awkwardly in this sentence to me, but I'm not sure what to do about it. Maybe it's that being (self-)assured we're doing things "in their best interest" sounds vaguely condescending? Perhaps a nod to user research would help here.